aws-cloud-account/track.yml

```diff
@@ -23,4 +23,3 @@ developers:
 - vadym@instruqt.com
 private: false
 published: true
-checksum: "13254736193028753480"
```

Comment:
We should also remove the ID from the `track.yml`, the CLI will resolve the correct ID through the track's slug. This way we can sync the tracks to both the DEV and PROD environment.

Reply:
I think we deliberately didn't wanna do the dev/prod separation here, as this is mainly meant to show clients how you could manage version control and CD for tracks.